pkg/install/install.go

```diff
@@ -38,7 +38,7 @@ func FillInTemplates(params TemplateParameters) (map[string][]byte, error) {
 		if info.IsDir() {
 			return nil
 		}
-		if (params.GitReadOnly || !params.RegistryScanning) && strings.Contains(info.Name(), "memcache") {
+		if params.RegistryDisableScanning && strings.Contains(info.Name(), "memcache") {
```

Comment:
The comment after suggests that memcache will be omitted if --git-readonly, which I think is incorrect. Either the comment needs changing, or the comment and the code ..

Reply:
It's the comment

Reply:
And this is why I missed it

<img width="833" alt="Screen Shot 2020-02-04 at 5 07 13 PM" src="https://user-images.githubusercontent.com/2362916/73762827-edd09300-4770-11ea-9297-2a2cac4e2548.png">

Goland collapses`if` blocks only including a return statement by by default (which I think is reasonable) but it also hides any comments inside them.